SimCalorimetry/Configuration/python/ecalDigiSequence_cff.py

```diff
@@ -18,8 +18,14 @@
 premix_stage1.toReplaceWith(ecalDigiTask, ecalDigiTask.copyAndExclude([simEcalPreshowerDigis]))
 
 from SimCalorimetry.EcalEBTrigPrimProducers.ecalEBTriggerPrimitiveDigis_cff import *
-_phase2_ecalDigiTask = ecalDigiTask.copy()
-_phase2_ecalDigiTask.add(simEcalEBTriggerPrimitiveDigis)
+
+#DARIO COMMENTED:
+#_phase2_ecalDigiTask = ecalDigiTask.copy()
+#_phase2_ecalDigiTask.add(simEcalEBTriggerPrimitiveDigis)
+
+#DARIO ADDED:
+#_phase2_ecalDigiTask = cms.Task(simEcalDigis)
+_phase2_ecalDigiTask = cms.Task()
```

Comment:
this looks like it's missing something

Reply:
we actually don't have the digi-to-raw for phase2 yet, the format hasn't been defined

Reply:
@kpedro88 can this PR be merged ? We don't have a digi-to-raw  step and trigger emulator, which are our next goal in the development of Phase2 Ecal Software. Having the Sim part (this PR) merged would help us tremendously

Reply:
It wasn't clear to me that the updates were finished. I will go through the code again.

Reply:
but at minimum, you need to apply `scram b code-checks` as indicated in https://github.com/cms-sw/cmssw/pull/29386#issuecomment-653462321